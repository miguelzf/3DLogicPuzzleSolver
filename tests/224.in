- C - - X -
- - - - - -
- P - Y - -
- - - - - X
B - X G - -
- - - - - -

- - G - X X
X - X - - -
- - - - - X
- - Y - - X
- X X - - -
- - - - - -

P - X - - -
- - - - X -
C - X - B -
X - - - - -
- - - - - -
- - - - - -

