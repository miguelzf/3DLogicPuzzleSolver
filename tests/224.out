
Solution:
-  C  c  -  X  -  
-  -  c  y  y  y  
-  P  c  Y  c  c  
-  p  c  c  c  X  
B  p  X  G  g  g  
b  p  p  p  p  p  
================
b  b  G  g  X  X  ||  P  g  X  c  y  y  
X  b  X  g  g  g  ||  g  g  -  c  X  y  
-  b  b  b  b  X  ||  C  c  X  c  B  y  
y  y  Y  -  b  X  ||  X  c  c  c  b  y  
y  X  X  -  b  b  ||  b  b  b  b  b  y  
y  y  y  y  y  y  ||  y  y  y  y  y  y  
